content-src/components/Search/Search.jsx

```diff
@@ -119,33 +118,32 @@ export class _Search extends React.PureComponent {
       {!this.props.handoffEnabled &&
       <div className="search-inner-wrapper">
         <label htmlFor="newtab-search-text" className="search-label">
-          <span className="sr-only"><FormattedMessage id="search_web_placeholder" /></span>
+          <span className="sr-only" data-l10n-id="newtab-search-box-search-the-web-text" />
         </label>
         <input
           id="newtab-search-text"
+          data-l10n-id="newtab-search-box-search-the-web-input"
           maxLength="256"
-          placeholder={this.props.intl.formatMessage({id: "search_web_placeholder"})}
           ref={this.onInputMount}
-          title={this.props.intl.formatMessage({id: "search_web_placeholder"})}
           type="search" />
         <button
           id="searchSubmit"
           className="search-button"
-          onClick={this.onSearchClick}
-          title={this.props.intl.formatMessage({id: "search_button"})}>
-          <span className="sr-only"><FormattedMessage id="search_button" /></span>
+          data-l10n-id="newtab-search-box-search-button"
+          onClick={this.onSearchClick}>
+          <span className="sr-only" data-l10n-id="newtab-search-box-search-text" />
```

Comment:
You don't need a span here, just use the value on the `..-search-button`.

Reply:
Not quite.. or not exactly. This `sr-only` class makes the text invisible except to screen readers, so either we make the fluent string value `…-search-button = <span data-l10n-name="sr-only">Search</span>` which seems unnecessarily complex when... we now have https://bugzilla.mozilla.org/show_bug.cgi?id=1556039 to clean up sr-only usage. In this specific case, I believe we can just get rid of the child span as the `title` attribute already makes it show up to screen readers and doesn't need a different `aria-label`:
![image](https://user-images.githubusercontent.com/438537/58722638-c547fc00-838c-11e9-9a3c-9418d42b9f84.png)